pkg/ebpf/c/tracee.bpf.c

```diff
@@ -2494,7 +2498,12 @@ int tracepoint__sched__sched_process_exec(struct bpf_raw_tracepoint_args *ctx)
 
     // We passed all filters (in should_trace()) - add this pid to traced pids set
     bpf_map_update_elem(&traced_pids_map, &data.context.host_tid, &data.context.host_tid, BPF_ANY);
-    bpf_map_update_elem(&process_context_map, &data.context.host_tid, process, BPF_ANY);
+
+    process_context_t *process = bpf_map_lookup_elem(&process_context_map, &data.context.host_tid);
+    if (process != NULL) {
+        __builtin_memcpy(process->comm, data.context.comm, TASK_COMM_LEN);
+        bpf_map_update_elem(&process_context_map, &data.context.host_tid, process, BPF_ANY);
```

Comment:
I don't think that it is required to do this map update.
You have a pointer to process, which updates the entry without a required update.
Please check.